public/manifest.json

```diff
@@ -10,16 +10,16 @@
     {
       "src": "logo192.png",
```

Comment:
This is the logo I am talking about

Reply:
AH! Good catch, I didn't notice that. Thanks